tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -1,510 +1,610 @@
 {
```

Comment:
<p><span class="mention" data-value="MatthieuMoreau0" data-denotation-char="@">﻿<span contenteditable="false">@MatthieuMoreau0</span>﻿</span> <span class="mention" data-value="MohamedAminMallek" data-denotation-char="@">﻿<span contenteditable="false">@MohamedAminMallek</span>﻿</span> Is the existence of <code>FRESNEL_RD</code> something we want to share / are ok with sharing with the world? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/952/discussion/#comment-2444241317'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>